manifests/a/ArobasMusic/GuitarPro/7/Soundbanks/1.1.123/ArobasMusic.GuitarPro.7.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ArobasMusic.GuitarPro.7.Soundbanks
+PackageVersion: 1.1.123
+Platform:
+  - Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: ArobasMusic.GuitarPro.7
```

Comment:
Won't this install the same package twice, since the InstallerURL of ArobasMusic.GuitarPro.7 is the same as this one?

Reply:
Probably. I'll remove that.